model-dmdocument/src/main/java/gov/nasa/pds/model/plugin/DMDocument.java

```diff
@@ -467,17 +453,47 @@ public static void main(String args[]) throws Throwable {
     // the use of the option "V" (alternate IM version) will change the input file directory (config
     // included)
     processArgumentParserNamespacePhase1(argparse4jNamespace);
-
-    // first get the environment variables
-    getEnvMap();
-    dataDirPath = lPARENT_DIR + "/Data/";
-
-    // if this is an LDDTool run then an alternate path is allowed (option "V")
-    if (LDDToolFlag && alternateIMVersionFlag) {
-      if (alternateIMVersion.compareTo(buildIMVersionFolderId) != 0) {
-        dataDirPath = lPARENT_DIR + "/Data/" + alternateIMVersion + "/";
+    
+    String sysDataHome = System.getProperty("data.home");
+//    System.out.println ("debug DMDocument HERE");
+//    System.out.println ("debug DMDocument -0- sysDataHome:" + sysDataHome);
+
+    if (sysDataHome != null) {
+//      System.out.println ("debug DMDocument -1- sysDataHome:" + sysDataHome);
+      sysDataHome = sysDataHome.replace('\\', '/');
+      lPARENT_DIR = sysDataHome + "/";

```

Comment:
@jshughes since this isn't a global constant, can we stick to using Java variable naming code conventions here instead (e.g. `parentDir` or something like that?)